kong/clustering/version_negotiation/init.lua

```diff
@@ -354,4 +354,40 @@ function _M.get_negotiated_service(name)
   return val.version, val.message
 end
 
+--- detect '/v1/wrpc' endpoint
+--- if there is no '/v1/wrpc', fallback to websocket + json
+function _M.head_request_version(conf, cert, cert_key)
+  local params = {
+    scheme = "https",
+    method = "HEAD",
+
+    ssl_verify = false,
```

Comment:
Seems to be a bad idea.

Reply:
In the first version check progress, We need not to verify cert, or else the test case in `06-version_negotiation_spec.lua` will fail, because it use a self-signed cert.